src/components/editMeasure/testCases/components/editTestCase/qiCore/EditTestCase.tsx

```diff
@@ -83,6 +83,7 @@ import useFhirCqlParsingService from "../../../api/cqlElmTranslationService/useF
 import checkSpecialCharacters from "../../../util/checkSpecialCharacters";
 import EditorSearch from "./LeftPanel/EditorSearch";
 import useFormikResetOnEvent from "../../../../../common/useFormikResetOnEvent";
+import moment from "moment";
```

Comment:
not recommended to use this anymore. can we use some other library? I think we already have dayjs or date-fns

Reply:
Good to know. When I looked it up people were saying to use moment which we also have in other places. I switched it to dayjs.

Reply:
its probably coming via cqm-execution or cqm-models